services/movie_session.py

```diff
@@ -41,4 +41,16 @@ def update_movie_session(
 
 
 def delete_movie_session_by_id(session_id: int) -> None:
-    MovieSession.objects.get(id=session_id).delete()
+    try:
+        MovieSession.objects.get(id=session_id).delete()
+    except MovieSession.DoesNotExist("There is no movie session with this ID"):
+        pass
```

Comment:
It seems like you are trying to pass a message to the `DoesNotExist` exception, but it doesn't work this way. This exception does not take any arguments. You can create a custom message and raise a new exception if `DoesNotExist` is caught.